scripts/bump_version.py

```
@@ -19,20 +20,224 @@
+    >>> data = {"package": {"version": "0"}}
+    >>> _update_package_version(data, "1")
+    >>> data["package"]["version"]
+    '1'
+    """
+    if "workspace" in doc and "package" in doc["workspace"]:
+        doc["workspace"]["package"]["version"] = version
+    elif "package" in doc:
+        doc["package"]["version"] = version
+
+
+def _extract_version_prefix(entry) -> str:
```

Ruff (0.12.2)

Missing type annotation for function argument `entry`

(ANN001)